controllers/utils.go

```diff
@@ -25,11 +25,13 @@ import (
 // extractNodeSelectorAndTolerations combines and extracts scheduling parameters from the multiple deviceClass entries in an lvmCluster
 func extractNodeSelectorAndTolerations(lvmCluster *lvmv1alpha1.LVMCluster) (*corev1.NodeSelector, []corev1.Toleration) {
 	var nodeSelector *corev1.NodeSelector
-	var tolerations []corev1.Toleration
+
+	tolerations := lvmCluster.Spec.Tolerations
```

Comment:
how about just returning `lvmCluster.Spec.Tolerations` from this method and not saving it in a variable first. The variable is never used.   

Or just have `extractNodeSelector` method. We can directly use `lvmCluster.Spec.Tolerations` if possible.

Reply:
While that is a valid comment,I would like to address that in a separate PR